.github/workflows/validate-issues.yaml

```diff
@@ -1,9 +1,56 @@
-name: add-issues-labels
+name: validate-issues
 on:
   issues:
     types: [opened, edited, reopened]
-
 jobs:
+  title-check:
+    runs-on: ubuntu-latest
+    env:
+      BODY: ${{ github.event.issue.body }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        persist-credentials: false
+        sparse-checkout: |
+          .github/scripts/pr-issue-info/issue-fail.md
+          .github/scripts/pr-issue-info/get_title_types.py
+          .github/issue-title-types.yaml
+    - name: Set up Python 
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
+    - name: Install dependencies
+      run: python3 -m pip install --upgrade pip pyyaml
+    - name: Check issue title 
+      env:
+        FILE_PATH: .github/issue-title-types.yaml
+      run: |
+        regex=$(python3 .github/scripts/pr-issue-info/get_title_types.py)
+        echo "Title regex: $regex"
+        title="${{ github.event.issue.title }}"
+        echo "$title" | grep -Pq "$regex" || (echo "$ERROR_MSG" && echo "TITLE_CHECK_FAILED=true" >> $GITHUB_ENV)
+    - name: Delete comment if title is fixed
+      if: env.TITLE_CHECK_FAILED != 'true'
+      uses: thollander/actions-comment-pull-request@v2
+      with:
+        message: |
+          Deleting comment, please refresh the page...
+        comment_tag: title_check
+        mode: delete
+        GITHUB_TOKEN: ${{ secrets.KICS_BOT_PAT }}
+    - name: Add comment if title fails
+      if: env.TITLE_CHECK_FAILED == 'true'
+      uses: thollander/actions-comment-pull-request@v2
+      with:
+        filePath: .github/scripts/pr-issue-info/issue-fail.md
+        comment_tag: title_check
+        mode: recreate
+        create_if_not_exists: true
+        GITHUB_TOKEN: ${{ secrets.KICS_BOT_PAT }}
+    - name: Fail the workflow
```

Comment:
Hi @freitasmillena.
This name sounds a bit awkward. Could you change it to "Workflow failed" (for example)?